src/abstractdataframe/join.jl

```diff
@@ -347,12 +347,12 @@ end
 
 """
     innerjoin(df1, df2; on = Symbol[], makeunique = false,
-              indicator = nothing, validate = (false, false))
+              validate = (false, false))
     innerjoin(df1, df2, dfs...; on = Symbol[], makeunique = false,
               validate = (false, false))
 
-Inner join two or more data frame objects and return a `DataFrame` containing
-the result.
+Perform an inner join two or more data frame objects and return a `DataFrame` containing
+the result. Only include rows with keys that match in all passed data frames.
```

Comment:
Written like this, it's not obvious that it's the definition an inner join. Maybe say "An inner join includes..." Same for other kinds.

Reply:
ok - all suggestions applied